.github/dependabot.yml

```diff
@@ -11,3 +12,31 @@ updates:
       prefix: "build(deps):"
       prefix-development: "build(deps-dev):"
     versioning-strategy: "increase"
+    groups:
+      dev-deps:
+        dependency-type: "development"
+        applies-to: "version-updates"
+        update-types:
+        - "minor"
+        - "patch"
+    groups:
```

Comment:
@petertonysmith94  I think we'd have to move the `dev-deps` config below line 22 in this case, otherwise applying this suggestion would apply single pull request `minor` and `patch` updates to dev dependencies.

Reply:
@maschad exactly. Outlined in the PR description but I decided to group dev deps together for `patch` and `minor`, and prod deps to just `patch`. Up for discussion though.

Reply:
Sorry @petertonysmith94 I do realise you were also mentioned `groups` was duplicated, which I have removed now 👍🏻